README.md

```diff
@@ -95,12 +95,15 @@ Like any open-source software, Arlo welcomes suggested changes in the form of pu
 2. Download [`pip`](https://pypi.org/project/pip/)
 3. Install `pipenv` (note: run `python3 -m pip install pipenv` to get a version that's compatible with your local python install if your system defaults to a python other than >3.7).
 4. Install [`yarn`](https://yarnpkg.com/en/docs/install).
-5. Install [`postgresql-dev`](https://www.postgresql.org/download).
-6. Install dependencies with `make install`
-7. Run via `./run-dev.sh`
+5. Install `postgres-client` and `postgresql-dev`, see https://www.postgresql.org/download.
+6. Install dependencies with `make install` or `make install-development` depending on your use-case
+7. Create a database config by copying `config/database.cfg.example` to `config/database.cfg`
+8. Run tests via `pipenv run python -m pytest tests/`
```

Comment:
We've got `make test-server` that handles this magic incantation for you.

Reply:
:eyes: will fix!